ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -238,11 +244,16 @@ def delete_workload(self, force=False):
             log.info("Verify backend RBD images are deleted")
             for cluster in get_non_acm_cluster_config():
                 config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+                rbd_pool_name = (
+                    (config.ENV_DATA.get("rbd_name") or rbd_name)
+                    if config.DEPLOYMENT["external_mode"]
+                    else constants.DEFAULT_CEPHBLOCKPOOL
```

Comment:
```suggestion
                    else rbd_pool_name
```

But why replacing `constants.DEFAULT_CEPHBLOCKPOOL` with `rbd_pool_name` is needed overall?

Reply:
ack